src/dosext/dpmi/dpmi.c

```diff
@@ -1053,10 +1053,12 @@ static inline int do_LAR(us selector)
   return ret;
 }
 
-int GetDescriptor(us selector, unsigned int *lp)
+int GetDescriptor(us selector, unsigned int *addr, dosaddr_t daddr)
```

Comment:
OK, that sucks, of course. :)
For the first time, please just make a
local buffer `unsigned int lp[2];` at 0x000b
call and do memcpy_2dos() from there, or
invalidate directly.
Both ways are quite ugly, but at least not
to mangle the public API.

Slightly better approach can be to
introduce SEL_ADR_W() macro that
will take also len and do the invalidation,
whereas the normal SEL_ADR() will
return const pointer. Eventually the
same can be done with LINEAR2UNIX(),
but that would likely be a lot of work.

Reply:
Done the first approach, as it's quite simple.